site/src/pages/WorkspacesPage/BatchDeleteConfirmation.tsx

```diff
@@ -264,10 +263,8 @@ const Resources: FC<StageProps> = ({ workspaces }) => {
 };
 
 const PersonIcon: FC = () => {
-	// This size doesn't match the rest of the icons because MUI is just really
-	// inconsistent. We have to make it bigger than the rest, and pull things in
-	// on the sides to compensate.
-	return <PersonOutlinedIcon css={{ width: 18, height: 18, margin: -1 }} />;
+	// Using the Lucide icon with appropriate size class
+	return <UserIcon className="size-icon-sm" css={{ margin: -1 }} />;
```

Comment:
```suggestion
	return <UserIcon className="size-icon-xs" />;
```

See my other comment in BatchUpdateConfirmation.tsx